schema/dereferenced-release-schema.json

```
@@ -6481,6 +6481,21 @@
             "description": "The single amendment object has been deprecated in favour of including amendments in an amendments (plural) array.",
             "deprecatedVersion": "1.1"
           }
+        },
+        "finalStatus": {
+          "title": "Tender final status",
+          "description": "The final status of the tender, from the closed [tenderFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#tender-final-status) codelist.",
```

```suggestion
          "description": "The final status of the tender, using the closed [tenderFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#tender-final-status) codelist.",
```
as per style guide https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#codelists

---

Done in https://github.com/open-contracting/standard/pull/1648/commits/4bab921b7905faf32c1a52101b733deb20ea7eb5